cloudflare.go

```diff
@@ -60,10 +61,17 @@ func (api *API) makeRequest(method, uri string, params interface{}) ([]byte, err
 	// req.Header.Add("Content-Type", "application/json")
 	client := &http.Client{}
 	resp, err := client.Do(req)
-	// TODO(jamesog): Check resp.StatusCode != http.StatusOK
-	// if resp.StatusCode != http.StatusOK
 	defer resp.Body.Close()
 	resBody, err := ioutil.ReadAll(resp.Body)
+	if resp.StatusCode != http.StatusOK {
```

Comment:
According to my testing where I entered invalid credentials, the body was still set. I don't know if we have a good way of testing against some endpoint which has an empty body, but would assume we at least always can read an empty string from the response?